### PR TITLE
Frontend new structure dummywebsite

### PR DESCRIPTION
**Pull request by aislinggallagher**
Vue basic struct with simple CI

### CODE REVIEW

**Comment by aislinggallagher** [comment]
Moved .github into same directory - was causing workflow to fail

File: dummysite-auth-module/.github/workflows/push.yml
[the remark this answers is not in the document]
Comment (DarraghOF) [reply dummysite-auth-module/.github/workflows/push.yml:1]: I see why you moved it here but it's not being picked up by github actions. There doesn't seem to be away to configure this action ` xRealNeon/VuePagesAction@1.0.1` to run a package.json from another folder.

For now I'd just remove this action. We can find a better solution in the future.

I would add a CI action that runs the build, lint and test:unit scripts from packages.json

File: dummysite-auth-module/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "build": "run-p type-check build-only",
+    "preview": "vite preview",
+    "test:unit": "vitest",
+    "build-only": "vite build",
+    "type-check": "vue-tsc --noEmit -p tsconfig.vitest.json --composite false",
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
+    "format": "prettier --write src/"
+  },
+  "dependencies": {
+    "vue": "^3.2.47"
```
Comment (DarraghOF) [inline dummysite-auth-module/package.json:16]: Are we going to install [vuetify ](https://vuetifyjs.com/en/) here? I know this is the dummy website so maybe you want to go with a different component library or no componenet library to make it look different to the main site but if you do want to go with vuetify there is a quick way to scaffold a new site using `npm init vuetify@latest`

File: dummysite-auth-module/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  "devDependencies": {
+    "@rushstack/eslint-patch": "^1.2.0",
+    "@types/jsdom": "^21.1.0",
+    "@types/node": "^18.14.2",
+    "@vitejs/plugin-vue": "^4.0.0",
+    "@vitejs/plugin-vue-jsx": "^3.0.0",
```
[the remark this answers is not in the document]
Comment (aislinggallagher) [reply dummysite-auth-module/package.json:28]: The vitejs plugin?

**Comment by mcarmenjc** [comment]
I think by error you added admin-site/node_modules to the changes. Can you please delete that folder?

File: dummysite-auth-module/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (DarraghOF) [reply dummysite-auth-module/package.json]: Yeah this one `@vitejs/plugin-vue-jsx`

**Comment by aislinggallagher** [comment]
@mcarmenjc @DarraghOF believe issues are fixed, PTAL

**Review by DarraghOF — APPROVED** [review]

**Review by mcarmenjc — APPROVED** [review]

**Pull request merged**